.github/workflows/image.yaml

```diff
@@ -297,6 +298,9 @@ jobs:
       - standard-upgrade-latest
       - test-uki
     steps:
+      - uses: actions/checkout@v3
+      - run: |
+          git fetch --prune --unshallow
```

Comment:
On another branch, we are fixing it like this: https://github.com/kairos-io/kairos/blob/81a5a9afe2d55b8ccbf5f9295ccac025b838ca15/.github/workflows/release.yaml#L15-L17